patches/chrome-common-importer-profile_import.mojom.patch

```diff
@@ -10,23 +10,27 @@ index 625b5ddec268292acf11f3c060c07c336ca1317f..1738d9d68a93ec1940688caa3352d55f
  import "url/mojom/url.mojom";
  
  const string kProfileImportServiceName = "profile_import";
-@@ -31,6 +32,9 @@ struct FaviconUsageDataList;
+@@ -31,6 +32,12 @@ struct FaviconUsageDataList;
  [Native]
  struct ImporterIE7PasswordInfo;
  
 +[Native]
```

Comment:
same as above, I think we can combine all Brave-specific data together

Reply:
but is this necessary for that to happen? Can't we still group it together even if it isn't all there?

Reply:
BraveExternalProcessImporterHost::StartImportSettings still gets the full list of selected items if we keep those split out

Reply:
and actually I thought this was a silent automatic import?

Reply:
> and actually I thought this was a silent automatic import?

We need to support both the silent automatic import case and the manual import case (i.e. former browser-laptop user downloads Brave-Core from website and installs it, then decides they want to import their old browser-laptop data). The importer code is identical between the use cases. The specifics of the silent automatic import case and handled in a separate PR (https://github.com/brave/brave-core/pull/729); among other things, it assumes the user wants to import all supported importer data types.